apps/client/src/config/index.ts

```diff
@@ -159,3 +159,9 @@ export const requiredFieldText: string = "Verplicht veld is niet ingevuld";
 export const requiredFieldRadio: string = "Maak een keuze";
 
 export const firstSelectOption: string = "Maak een keuze";
+
+export const urls: { [key: string]: string } = {
+  OMGEVINGSLOKET: "https://www.omgevingsloket.nl",
+  AMSTERDAM_MORE_INFO:
```

Comment:
Please make this more specific, because all pages of Amsterdam are used for more information. Something like: HOW_TO_GET_A_PERMIT